ccd-definition/CaseEvent/User/UserEvents-R2-CUI-nonprod.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "CLAIMANT_RESPONSE_CUI",
+    "Name": "Claimant Response",
+    "Description": "Claimant response to defendant from Citizen",
+    "DisplayOrder": 6,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
```

Comment:
if we are changing the status of the claim we should put it here

Reply:
yes.. something like this 
PostConditionState: "JUDICIAL_REFERRAL" and remove it from civil-service handler.

Reply:
done